lib/builderator/interface/packer.rb

```diff
@@ -63,24 +63,10 @@ def initialize(*_)
             json[:builders] << build_hash
           end
 
-          post_processors = []
-
-          # Post-processors should be considered as a sequence
-          Config.profile.current.packer.post_processor.each do |name, post_processor|
-            post_processor_hash = post_processor.to_hash
-
-            # Single, named step in a sequence
-            if post_processor_hash.empty?
-              post_processors << name
-              next
-            end
-
-            # The post-processor's type should be the same as the name
-            post_processor_hash[:type] = name
-
-            post_processors << post_processor_hash
-          end
-          json['post-processors'].push(post_processors)
+          # post_processors must be a List of Rashes.
+          post_processors = Config.profile.current.packer.post_processors.to_a
+          json['post-processors'] = post_processors.first unless post_processors.empty?
```

Comment:
This isn't a case where there's a single element. Unfortunately [`List.set`]() pushes the elements being set onto itself so the result of pushing an array onto another list without flattening (see [L28](https://github.com/rapid7/builderator/pull/99/files#diff-600bb13a6156e01bcea7563047d0ab1fR28)) is a single element array containing the full array of values.